### PR TITLE
Earth and Eastuary in harmony

### PR DESCRIPTION
**Pull request by garnetgrimm**
The `json2daisy` template generator was broken to begin with the fact that the daisy seed and patch sm API's are entirely different, I've come to  the conclusion that it would take significantly more work to fix than its worth. Using `json2daisy` is does not offer enough flexibility for our use case, and we don't even really get any benefit from using it other than it is kind of cool and we could re-use the existing json files. 

TLDR I hand wrote initialization code for each of the modules, and it was significantly easier than getting `json2daisy` working properly. Plus the interface is identical between them :)

### CODE REVIEW

**Comment by hypergrace** [comment]
Fantastic !! I look forward to diving into this Tuesday. Thank you for this work -- so exciting!

**Review by hypergrace — COMMENTED** [review]
Very impressive. I like the synchrony between these platforms -- again super impressive!!
